.github/workflows/go.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.15', '1.16', '1.17']
```

Comment:
Yea, that's sort of what the COPR integration was doing at one point. It would make an RPM using the .copr/Makefile and .copr/yggdrasil.spec.in files. It looks like I've since taken it down though. I'm not fond of the idea of keeping distro-specific build files in the upstream repository. The way it is packaged in Fedora differs from CentOS and RHEL. Upstream should be distribution-agnostic.